packages/firestore/android/src/main/java/io/invertase/firebase/firestore/UniversalFirebaseFirestoreCommon.java

```diff
@@ -79,7 +85,7 @@ private static void setFirestoreSettings(FirebaseFirestore firebaseFirestore, St
 
     firebaseFirestore.setFirestoreSettings(firestoreSettings.build());
 
-    settingsLock.put(appName, true);
+    preferences.clearAll();
```

Comment:
This needs to be per key - if other plugins use shared prefs it'll also remove these.